couchdbkit/client.py

```diff
@@ -752,11 +753,16 @@ def copy_doc(self, doc, dest=None, headers=None):
         return {'ok': False}
 
     def raw_view(self, view_path, params):
-        if 'keys' in params:
-            keys = params.pop('keys')
-            return self.res.post(view_path, payload={ 'keys': keys }, **params)
+        params.pop('dynamic_properties', None)
+        if view_path == '_all_docs':
+            return self.cloudant_database.all_docs(**params)
         else:
-            return self.res.get(view_path, **params)
+            view_path = view_path.split('/')
+            assert len(view_path) == 4
+            ddoc = DesignDocument(self.cloudant_database, view_path[1])
+            ddoc.fetch()
+            view = ddoc.get_view(view_path[3])
+            return view(**params)
```

Comment:
Adding an assert here seems strange when it would fail without changing the db 3 lines later. Also it's not super clear to me that this does the same thing as before. THey look like wildly different functions

Reply:
What if we replaced the assert with ```assert len(view_path) == 4```?

Reply:
(edited)